### PR TITLE
added 403 for overview page

### PR DESCRIPTION
**Pull request by Gerwoud**

### CODE REVIEW

File: frontend/src/loaders/submission-overview-loader.ts [hunk 1]
```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
 }: {
   params: Params<string>;
 }) {
+
```
Comment (AronBuzogany) [inline frontend/src/loaders/submission-overview-loader.ts:30]: unnecessary new line

File: frontend/src/loaders/submission-overview-loader.ts [hunk 2]
```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
...
   params: Params<string>;
 }) {
+
   const projectId = params.projectId;
   const projectResponse = await authenticatedFetch(
     `${APIURL}/projects/${projectId}`
   );
+
+  if(projectResponse.status != 200) {
+    throw new Response("Not authenticated", {status: 403});
```
Comment (AronBuzogany) [inline frontend/src/loaders/submission-overview-loader.ts:37]: You can simply return the response, this is very unnecessary

File: frontend/src/loaders/submission-overview-loader.ts [hunk 3]
```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
 }: {
   params: Params<string>;
 }) {
+
   const projectId = params.projectId;
   const projectResponse = await authenticatedFetch(
     `${APIURL}/projects/${projectId}`
   );
+
+  if(projectResponse.status != 200) {
```
Comment (AronBuzogany) [inline frontend/src/loaders/submission-overview-loader.ts:36]: This isn't the correct way of checking whether the response is ok

**Review by AronBuzogany — CHANGES_REQUESTED** [review]

File: frontend/src/loaders/submission-overview-loader.ts (hunk 1 above)
Comment (Gerwoud) [reply frontend/src/loaders/submission-overview-loader.ts:30]: It adds to the appearance off the codebase

File: frontend/src/loaders/submission-overview-loader.ts (hunk 2 above)
Comment (Gerwoud) [reply frontend/src/loaders/submission-overview-loader.ts:37]: No it doesn't work then.

File: frontend/src/loaders/submission-overview-loader.ts (hunk 3 above)
Comment (Gerwoud) [reply frontend/src/loaders/submission-overview-loader.ts:36]: Then what is?

File: frontend/src/loaders/submission-overview-loader.ts (hunk 1 above)
Comment (AronBuzogany) [reply frontend/src/loaders/submission-overview-loader.ts:30]: You should format the document with prettier for this! Appearance is something subjective. To keep our code base maintainable and readable by all developers its best we keep ourselves to the conventions and not to our own subjective preferences.

Comment (Gerwoud) [reply frontend/src/loaders/submission-overview-loader.ts:30]: No it isn't subjective cause my opinion is absolute.

File: frontend/src/loaders/submission-overview-loader.ts (hunk 2 above)
Comment (AronBuzogany) [reply frontend/src/loaders/submission-overview-loader.ts:37]: Then return a HTTP response

File: frontend/src/loaders/submission-overview-loader.ts (hunk 3 above)
Comment (AronBuzogany) [reply frontend/src/loaders/submission-overview-loader.ts:36]: @Gerwoud You are a JS developer, you need to be able to do this research yourself. It's `projectRespone.ok`

Comment (Gerwoud) [reply frontend/src/loaders/submission-overview-loader.ts]: https://github.com/SELab-2/UGent-3/pull/414/commits/c36f3d74631884a359636dd5dc7ec177d1973124

**Review by Vucis — APPROVED** [review]

**Pull request merged**